config/index.js

```diff
@@ -26,6 +27,11 @@ function parseRoleFromClaim(claim) {
   return match ? match[1] : null;
 }
 
+function parseEmailFromClaim(claim) {
+  // input format example: "test@example.com:OVO001827"
+  return claim.split(':')[0];
```

Comment:
I don't think we're ever going to have an issue with this in practice, so we can leave it as is, but technically [`:` can be part of an email address](https://en.wikipedia.org/wiki/Email_address#Local-part).

Reply:
Check. I considered an rsplit, but I'm unsure how the `vo_email` claim looks like for people that have a relation to multiple OVO. That would be more likely to break with an `rsplit`.